src/ReactiveUI.SourceGenerators/ObservableAsProperty/ObservableAsPropertyGenerator{FromObservable}.Execute.cs

```diff
@@ -144,8 +149,31 @@ private static string GenerateObservableSource(string containingTypeName, string
 
 namespace {{containingNamespace}}
 {
+    {{parentClassDeclarationsString}}{{classes}}{{closingBrackets}}
+}
+#nullable restore
+#pragma warning restore
+""";
+    }
+
     /// <summary>
-    /// Partial class for the {{containingTypeName}} which contains ReactiveUI Reactive property initialization.
+    /// Generates the source code.
+    /// </summary>
+    /// <param name="containingTypeName">The contain type name.</param>
+    /// <param name="containingNamespace">The containing namespace.</param>
+    /// <param name="containingClassVisibility">The containing class visibility.</param>
+    /// <param name="containingType">The containing type.</param>
+    /// <param name="properties">The properties.</param>
+    /// <returns>The value.</returns>
+    private static string GenerateClassWithProperties(string containingTypeName, string containingNamespace, string containingClassVisibility, string containingType, ObservableMethodInfo[] properties)
+    {
+        // Includes 2 tabs from the property declarations so no need to add them here.
+        var propertyDeclarations = string.Join("\n\r", properties.Select(GetPropertySyntax));
```

Comment:
`\r\n` or `\n\r`

Reply:
was supposed to be just `\n` as per the others